types/read-pkg-up/index.d.ts

```diff
@@ -0,0 +1,31 @@
+// Type definitions for read-pkg-up 2.0
+// Project: https://github.com/sindresorhus/read-pkg-up
+// Definitions by: Louis Orleans <https://github.com/dudeofawesome>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace ReadPkgUp {
```

Comment:
Please export the members of this namespace as shown in [module-function.d.ts](http://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-function-d-ts.html).